createBookFromReadme.sh

```diff
@@ -49,7 +49,7 @@ function createSummary(){
 # Note:
 #     Install mdBook as per instructions in their repo https://github.com/rust-lang/mdBook
 function buildAndServeBookLocally(){
-    mdBook build && mdBook serve --open
+    mdbook build && mdbook serve --open
```

Comment:
After this fix: 

![Screenshot from 2021-01-04 17-03-56](https://user-images.githubusercontent.com/2096087/103554243-de8d2480-4eae-11eb-8b10-1d28f5b2ba4f.png)